site/constants/index.ts

```diff
@@ -53,7 +53,14 @@ export const COOKIES_DISRUPTION_INFO = "disruption-info";
 
 export const COOKIES_DISRUPTION_ERRORS = "disruption-errors";
 
+export const COOKIES_ADD_CONSEQUENCE_INFO = "add-consequence-info";
```

Comment:
```suggestion
export const COOKIES_ADD_CONSEQUENCE_INFO = "cdd-consequence-info";
```
Should all cookies start with cdd @lj-kpmg ?

Reply:
done